defaults/main.yml

```diff
@@ -17,7 +17,7 @@ ovirt_engine_setup_dwh_db_name: 'ovirt_engine_history'
 ovirt_engine_setup_dwh_db_user: 'ovirt_engine_history'
 ovirt_engine_setup_dwh_vacuum_full: false
 
-ovirt_engine_setup_firewall_manager: 'firewalld'
+ovirt_engine_setup_firewall_manager: '{{ default_firewall_manager }}'
```

Comment:
What is the reason for this change and why use vars/main.yml? In this way, the user cannot edit the variable from vars.

Reply:
This is exactly the purpose. We want user to override variable `ovirt_engine_setup_firewall_manager` which is indeed firewall manager of their choice. That is why `ovirt_engine_setup_firewall_manager` is in default/main.yml - because we **expect** users to override the default values.

However, no matter what will the user provide to `ovirt_engine_setup_firewall_manager`, we always want to treat `firewalld` as the default manager. This is something that user doesn't choose. That's why I put `default_firewall_manager` into vars/mail.yml.

Alternate approach would be to just keep it in default/main.yml. But then, in all the conditional comparisons, I would need to spell out "firewalld" explicitly. For example `when: ovirt_engine_setup_firewall_manager == firewalld`. This seems like a bad design from maintainability point of view.

Reply:
So if I understand it correctly the plan is to drop all options for `ovirt_engine_setup_firewall_manager` and always use `firewalld`? Because right now users can choose between `firewalld`, `iptables` or `null`.

Reply:
You could just skip the `ovirt_engine_setup_firewall_manager == firewalld` so it would check only if the package is in the facts. 
example:
```yml
      when: 
        - ovirt_engine_setup_firewall_manager not in ansible_facts.packages
        - ovirt_engine_setup_firewall_manager
```
this works when the `ovirt_engine_setup_firewall_manager` is  `null`, `firewalld` and `iptables`

You don't need to specify extra variables.